packages/account/src/Sections/Security/AccountLimits/account-limits.jsx

```diff
@@ -1,6 +1,6 @@
 import classNames from 'classnames';
 import React from 'react';
-import { Popover, DesktopWrapper, Loading, MobileWrapper, ThemedScrollbars } from '@deriv/components';
+import { Popover, DesktopWrapper, Loading, MobileWrapper, ThemedScrollbars, Text as TextComp } from '@deriv/components';
```

Comment:
Could we remove the alias?

Reply:
We have another Text component import, So I need to do this. Do you have better suggestion for name?

Reply:
It's safe to refactor all instances of `Text` from **packages/account/src/Components/text**  to our new `Text`.
Previous one is just a `<p>` wrapper. BTW your task is about spans.
So could you check who's responsible for p tags in account package so we can tell them to do this?

Reply:
I'm asking Sara on Slack

Reply:
I agree with Salar it's better if we use a similar component for all text